### DISCUSSION TITLE
Listener in remote chunking worker

### DISCUSSION THREAD

**Discussion by joethebatchguy** [spring-projects/spring-batch#4706] [Q&A]
We started to adopt remote chunking as our data volume keeps growing. It works great but with only one caveat: no listener anymore. 

We used to do a lot of logging and auditing using process listener and writer listener. However none of these are available in a remote chunking setup. 

Is there anyway to enable these listeners in a remote chunking worker?

**Comment by fmbenhassine** [comment]
Is the listener you are referring to defined on the manager side or the worker side? Not sure i fully get your setup and expectations. Please share the minimal code required to be able to understand your issue and help you efficiently.

**Reply by liuqiaowei512** [reply]
It's on the worker side. For example if we are using `RemoteChunkingWorkerBuilder` in the worker side, we cannot attach any listener at all
